src/engraving/dom/tremolosinglechord.h

```diff
@@ -166,29 +116,13 @@ class TremoloSingleChord final : public EngravingItem
     TremoloSingleChord(Chord* parent);
     TremoloSingleChord(const TremoloSingleChord&);
 
-    void setBeamPos(const PairF& bp);
-
-    TremoloType m_tremoloType = TremoloType::R8;
+    TremoloType m_tremoloType = TremoloType::INVALID_TREMOLO;
     Chord* m_chord1 = nullptr;
```

Comment:
Can't we remove this too, and just use `chord()`, i.e. the parent chord?

Reply:
There were difficulties, I wanted to keep it for later... I removed it, everything seems to be fine.